src/marqo/core/structured_vespa_index/common.py

```diff
@@ -1,7 +1,8 @@
 from marqo.core.models.marqo_index import DistanceMetric
 
 FIELD_ID = "marqo__id"
-FIELD_SCORE_MODIFIERS = 'marqo__score_modifiers'
+FIELD_SCORE_MODIFIERS_DOUBLE_LONG = 'marqo__score_modifiers_double_long'
```

Comment:
This is not backwards compatible. Older indexes won't have these fields and this needs to be handled